### ISSUE TITLE
Installing the necessary packages on the raspberry pi

### ISSUE THREAD

**Issue by ACovaci99** [ACovaci99/I_Love_Science_Fest#2]
See if the usage of a conda (or even miniconda) environment may help with installing the packages on the raspberry pi.
Also, have a look through the code and see what packages we would need. You can list them in this issue and update it if needed.

For now (12/07), we need:
- numpy 
- cv2
- time (optional)
- picamera
- PIL
- matplotlib
- pandas
- joblib
More will be added as the code progresses.

**Comment by FirassKooli** [comment]
I fixed the issues without using conda or miniconda as it wouldn't offer much advantages that way since our chip's CPU is ARM and conda is most aimed and supported for x86 architecture.

**Comment by hosseindehghanipour1998** [comment]
@FirassKooli @ACovaci99 
We need a package manager (miniconda preferably) to:

1. Keep track of the libraries and packages we have installed on the board (what we talked about today regarding keeping track of the packages).
2. Using the environment, we can extract all the packages and their versions simply using exactly one command and installing all of them on the rest of the raspberries using only one command (there is no need to install them all again one by one - one command will do it all for us).
3. If there is an urge to upgrade or downgrade a package to match another library later on while we progress, the package manager simply helps us do that way much easier. 

I suggested Miniconda or Anaconda because I am more familiar with that and if things go wrong it would be easier for us to fix.
